src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -696,6 +696,13 @@ export default {
 		onClick(event, navigate, routerLinkHref) {
 			// Always forward native event
 			this.$emit('click', event)
+
+			//If no links are defined, and collapsing is enabled, we want it to collapse
+			if (this.to === null && this.href === null && this.collapsible === true) {
+				this.open = !this.open
```

Comment:
(Never mutate a prop directly)

```suggestion
				this.toggleCollapse()
```

Reply:
Right